Add libimagequant to features.py

nulano opened the pull request:
From https://github.com/python-pillow/Pillow/pull/4084#issuecomment-535702633

Changes proposed in this pull request:

 * Add libimagequant to `features.py`
 * Remove code duplication from `selftest.py` introduced in #3870

radarhere commented on src/PIL/features.py:
```diff
@@ -133,37 +135,41 @@ def pilinfo(out=None):
         ("zlib", "ZLIB (PNG/ZIP)"),
         ("libtiff", "LIBTIFF"),
         ("raqm", "RAQM (Bidirectional Text)"),
+        ("libimagequant", "LIBIMAGEQUANT (quantization method)"),
```
```suggestion
        ("libimagequant", "LIBIMAGEQUANT (Quantization method)"),
```

nulano replied:
Done.

radarhere commented on src/PIL/features.py:
```diff
@@ -94,7 +95,7 @@ def get_supported():
 
 
-def pilinfo(out=None):
+def pilinfo(out=None, brief=False):
```
Suggestion: Instead of 'brief=False', what about 'include_python_version=True'?

hugovk replied:
Or flip it and call it `verbose`?

nulano replied:
@radarhere The significant difference is whether the list of supported image formats is printed.

Thinking about it, I don't think it would be a bad idea to include the Python version in `selftest.py`. The parameter could then be called `supported_formats=True`.

nulano replied:
> Thinking about it, I don't think it would be a bad idea to include the Python version in `selftest.py`. The parameter could then be called `supported_formats=True`.

Done, see comment below.

nulano commented:
I have renamed the `brief=False` parameter to `supported_formats=True`, and added the Python version to the `selftest.py` output.

The output is now (from https://travis-ci.org/nulano/Pillow/jobs/597002577#L2460):
```
python selftest.py
--------------------------------------------------------------------
Pillow 6.2.0.dev0
Python 3.7.1 (default, Dec  5 2018, 18:09:53)
       [GCC 5.4.0 20160609]
--------------------------------------------------------------------
Python modules loaded from /home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/Pillow-6.2.0.dev0-py3.7-linux-x86_64.egg/PIL
Binary modules loaded from /home/travis/.cache/Python-Eggs/Pillow-6.2.0.dev0-py3.7-linux-x86_64.egg-tmp/PIL
--------------------------------------------------------------------
--- PIL CORE support ok
--- TKINTER support ok
--- FREETYPE2 support ok
--- LITTLECMS2 support ok
--- WEBP support ok
--- WEBP Transparency support ok
--- WEBPMUX support ok
--- WEBP Animation support ok
--- JPEG support ok
--- OPENJPEG (JPEG2000) support ok
--- ZLIB (PNG/ZIP) support ok
--- LIBTIFF support ok
--- RAQM (Bidirectional Text) support ok
--- LIBIMAGEQUANT (Quantization method) support ok
--------------------------------------------------------------------
Running selftest:
--- 59 tests passed.
```

Running `python -m PIL` lists supported formats instead of the selftest message at the end, same as before.

hugovk commented:
Thanks!
